eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/EventHubClient.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.eventhubs;
+
+import com.azure.eventhubs.implementation.ClientConstants;
+
+// Each time a build method is called, a new receiver or sender is created.
+class EventHubClient implements AutoCloseable {
+    // Creates a new sender.
+    EventHubSender createSender() {
+        return new EventHubSender(ClientConstants.MAX_MESSAGE_LENGTH_BYTES);
```

Comment:
Do we want to support users overriding the max message length?

Reply:
No. This is a stub method. When this gets implemented, we'll connect to Event Hubs and then ask it for the connection's message capacity.